magento-devdocs/gatsby-doc-shell
# Spectrum theme

Converts site theme to closely match the devdocs theme.

TODO: Extract an actual importable Gatsby theme to use for other potential Gatsby projects

## Testing

1. Clone repo
1. Install dependencies
```sh
yarn install
```
3. Set up `.env` file with proper values (see example `.env` files)
1. Run dev server
```sh
yarn develop
```
5. Verify site builds and looks as expected
6. Build production site
```sh
yarn build
```
7. Run production server process
```sh
yarn serve
```
8. Verify site looks as expected
